drivers/SmartThings/matter-switch/fingerprints.yml

```
@@ -218,7 +218,7 @@ matterManufacturer:
     vendorId: 0x115A
     productId: 0x711
     deviceProfileName: light-color-level-2700K-6500K
- - id: "4442/72"
+  - id: "4442/72"
```

```suggestion
 - id: "4442/72"
```
nit: remove the extra space that was added here